src/Services/ConvertToQtiService.php

```diff
@@ -194,12 +199,15 @@ private function convertAssessmentItem($json)
         $content = $json['content'];
         $tags = $json['tags'];
         $itemReference = $json['reference'];
+
         foreach ($json['questions'] as $question) :
             $question['content'] = $content;
             $question['itemreference'] = $itemReference;
             if (in_array($question['data']['type'], LearnosityExportConstant::$supportedQuestionTypes)) {
                 $result = Converter::convertLearnosityToQtiItem($question);
                 $result[0] = str_replace('/vendor/learnosity/itembank/', '', $result[0]);
+                $result[0] = str_replace('xmlns:default="http://www.w3.org/1998/Math/MathML"', '', $result[0]);
+                $result[0] = str_replace('default:', '', $result[0]);
```

Comment:
Hi @rakeshqdsa ,
With regards to the string replacement of `default:`, is there a way that you can narrow down the affected scope of XML? 
We'd like to mitigate the risk of affecting any XML outside of mathML.
If not, could you please add a TODO comment above this line like: `TODO: Change this to only select MathML elements`?
At a minimum, we should add something like the following to make it specifically select the XML tags, and avoid affecting content inside the item body:
```
$result[0] = str_replace('<default:', '<', $result[0]);
$result[0] = str_replace('</default:', '</', $result[0]);
```

Can you also fix the following annotation?
```
@param  string $json    <- should be array not string at LINE 188
```

Reply:
Done